packages/ihe-gateway/scripts/init.sh

```diff
@@ -5,17 +5,17 @@ set -e
 
 source ./scripts/load-env.sh
 
-if [ -z "${CONFIG_BUCKET_NAME}" ]; then
-  echo "CONFIG_BUCKET_NAME is not set, skipping downloading certs and custom extensions."
-  exit 0
+if [ -z "${IHE_GW_CONFIG_BUCKET_NAME}" ]; then
+  echo "IHE_GW_CONFIG_BUCKET_NAME is not set, skipping downloading certs and custom extensions."
+  exit 1
```

Comment:
nit: if this is supposed to be an error case, should we change the `echo` to be more reflective of an error?

something like `Error: IHE_GW_CONFIG_BUCKET_NAME must be set to use this script`

Reply:
updated